### DISCUSSION TITLE
Amalgamation of different polarity peak list

### DISCUSSION THREAD

**Discussion by Lipidbender** [systemsomicslab/MsdialWorkbench#180] [General]
Does anyone have an experience with "Amalgamation of different polarity peak list" feature of MS-DIAL?
If, yes your help will be much appreciated :)
How do you prepare the *txt negative list file do you have an example?
Cheers.
Cagakan

**Comment by kozo2** [comment]
Hi @Cagakan31 
I believe the information on that topic is described in 
https://mtbinfo-team.github.io/mtbinfo.github.io/MS-DIAL/tutorial.html#section-5-12-2 .
If you still have questions, please feel free to ask again.
